app/main/routes.py

```diff
@@ -127,6 +134,51 @@ def poc_search():
     )
 
 
+@bp.route("/poc-browse-transferring-body", methods=["POST", "GET"])
+@access_token_sign_in_required
+def poc_browse_transferring_body():
```

Comment:
not reviewin this view since it will be superseded by the actual search view when integrating.

Reply:
poc-browse-transferring-body route will be redundant and superseded by front end browse view route